CHANGELOG.md

```diff
@@ -1,11 +1,27 @@
 # Upcoming
 
+# v8.0.0-beta
+
 ## Breaking changes
 
-- (>=v8.0.0) Setting model data for a `qx.ui.table.Table` when the table is still editing will
+- Setting model data for a `qx.ui.table.Table` when the table is still editing will
 now raise an error as this could have lead to an invalid edit. To prevent any errors, ensure
 that the table edits are completed or cancelled before refreshing table model data.
 
+- Properties and members are now in the same namespace. Formerly, a class could have a member variable and a property with the same name, and there was no conflict. Now, since properties are first-class and can be manipulated as normal members, the properties and members use the same namespace, so a single name can not be defined in both.
```

Comment:
Technically I don't think they are first class, just native properties - eg a function is called "first class" because it is an object of type `Function` which has properties, as well as having native functionality as a function

Reply:
I was using the terminology you used in https://github.com/qooxdoo/qooxdoo/issues/10410, but I just want it to be clear what the new feature does. I don't really care what term we use. Would you prefer that I change the docu to refer to them as native, rather than first-class?

Reply:
yes please; the terminolgoy in #10410 was referring to creating a `qx.Property` object that would describe the entire property (eg much like `Function` describes a function)

Reply:
Done